packages/@lightningjs/ui-components/src/components/Input/Input.mdx

```diff
@@ -61,7 +61,7 @@ class Example extends lng.Component {
 
 ### Setting/Inserting values
 
-The `Input` component keeps track of it's current `value` and cursor `position`. You can manage the value state in a few ways. Let's work off of the above example. We see the cursor blinking which is the `placeholder` value when listening mode is true and component in focused mode.
+"The `Input` component tracks its current value and cursor position, and provides different ways to manage the value state. When the listening mode is enabled, the cursor blinks as a `placeholder`, whether the component is in focused or unfocused mode."
```

Comment:
Why is this line in quotation marks?

Reply:
oops that was unintentional. Thanks for catching. I will remove them.